inc/Task.php

```diff
@@ -70,15 +59,12 @@ public function __construct(\helper_plugin_fkstaskrepo $helper, $year, $series,
         $this->helper = $helper;
     }
 
-    /**
-     * @param $figures
-     */
-    public function saveFiguresRawData($figures) {
+    public function saveFiguresRawData(array $figures): void {
         $this->figures = [];
 
         foreach ($figures as $figure) {
             $figureGoodForm = [];
-            $figureGoodForm['caption'] = $figure['caption'];
+            $figureGoodForm['caption'] = $figure['caption']; // TODO never used!!!
```

Comment:
Never used

Reply:
There are still some problems regarding figures that need to be solved. I plan to look at it soon...